baselines/getting_started/aws/aws_mods/variables.tf

```diff
@@ -0,0 +1,129 @@
+variable "mod_list" {
```

Comment:
This definition is in both the variables and the demo file? 
How does this work? 

Have you tried running the script against an environment to check that it works?

Reply:
I have changed the definition in demo and variables as below,

demo.tfvars has the example values.
variables.tf has the full mod list.